example/spec/helpers/kinesisHelpers.js

```diff
@@ -107,7 +107,7 @@ async function createKinesisStream(streamName) {
       try {
         return await kinesis().createStream({ StreamName: streamName, ShardCount: 1 });
       } catch (error) {
-        if (error.code === 'LimitExceededException') throw error;
+        if (error instanceof LimitExceededException) throw error;
```

Comment:
An alternative for all of this error handling is helper functions like "isLimitExceed" that we could export from aws-client instead of importing error types into packages like this. I didn't do that because it felt like complexity for the sake of it but open to pushback.